app/services/course_search_service.rb

```diff
@@ -11,7 +11,7 @@ def initialize(
 
   class << self
     def call(**args)
```

Comment:
side note we could probably leverage the new forwarding pattern to make this cleaner:

```
    def call(...)
      new(...).call
    end
```

Reply:
Done